apps/cli/src/node/docker-compose-validator.yaml

```diff
@@ -2,7 +2,8 @@ version: "3.9"
 
 services:
     validator:
-        image: sunodo/rollups-node:0.7.0
+        image: cartesi/rollups-node:1.3.1
+        user: root
```

Comment:
do we need to force root here?

Reply:
Cartesi/rollups-node uses the cartesi user

We need root to make the snapshot copy